src/Api/Controllers/AppsController.php

```diff
@@ -36,13 +50,37 @@ class AppsController extends BaseController
      */
     public function onConstruct()
     {
+        $random = new Random();
         $this->model = new Apps();
+        $this->model->key = $random->uuid();
```

Comment:
@rwhite27 move this to the beforeCreate function in the model since this only needs to run once